modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7542,6 +7542,10 @@ feature.flag.LPD-38869.description=Private pages are deprecated in favor of cont
 feature.flag.LPD-38869.title=Private Pages
 feature.flag.LPD-39304.description=The asset selection of asset publisher has been unified with collections and the configuration must be done with an existing one.
 feature.flag.LPD-39304.title=Asset Selection for Asset Publisher
+feature.flag.LPD-40533.description=Widget Setting provided by Liferay.
```

Comment:
There is a small chance Brian will say that this is not a full sentence. I'm not a native speaker, so it might be fine.

Reply:
This sentence was written as a proposal by the Product team, so we hope Brian approves it :crossed_fingers: